src/main/java/seedu/address/model/person/weblink/WebLinkUtil.java

```diff
@@ -0,0 +1,19 @@
+package seedu.address.model.person.weblink;
+
+/**
+ * Contains Command Line Interface (CLI) syntax definitions common to multiple commands
+ */
+public class WebLinkUtil {
+
+    /*Keywords that can be used to match website to certain categories. */
+    public static final String FACEBOOK_MATCH_STRING = "facebook.com";
```

Comment:
as per within the "Weblink" side, this could be handled with a regex instead of match string for more fine-tuned checking, such as allowing for fb.com or instagr.am links as well :)

Reply:
HMMM okok, i get. thanks!!